test/lib/rule-finder.js

```diff
@@ -2,10 +2,17 @@ const path = require('path');
 const assert = require('assert');
 const proxyquire = require('proxyquire');
 
+const semver = require('semver');
+const eslintPkg = require('eslint/package.json');
+
 let ModuleResolver;
 try {
-  // eslint 7.12+
-  ModuleResolver = require('@eslint/eslintrc/lib/shared/relative-module-resolver');
+  if (semver.satisfies(eslintPkg.version, '>= 7.12')) {
+    // eslint 7.12+
+    ModuleResolver = require('@eslint/eslintrc/lib/shared/relative-module-resolver');
```

Comment:
```suggestion
    { Legacy: ModuleResolver } = require('@eslint/eslintrc');
```

Reply:
oops, thanks, i had that change (plus parens) locally but lost it in a rebase.